src/sentry/grouping/api.py

```diff
@@ -75,7 +75,7 @@ def get_config_dict(self, project: Project) -> GroupingConfig:
             "enhancements": self._get_enhancements(project),
         }
 
-    def _get_enhancements(self, project):
+    def _get_enhancements(self, project) -> str:
```

Comment:
Can you also add a type to `rv` in line 92? We are returning it, but it is interpreted as `any`

Reply:
Additionally `Enhancements.from_config_string` also returns any and creates confusion.